xsd/netex_part_3/part3_salesTransactions/netex_fareDebit_version.xsd

```diff
@@ -0,0 +1,351 @@
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- ==== BOOKING DEBIT ================================================ -->
+	<xsd:element name="BookingDebit" abstract="false" substitutionGroup="FareDebit_DummyType">
+		<xsd:annotation>
+			<xsd:documentation>A LOG ENTRY recording data about a debiting action for a booking fee.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="BookingDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="BookingDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="BookingDebitIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="BookingDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for BOOKING DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="FareDebit_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="BookingDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="BookingDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for BOOKING DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence/>
+	</xsd:group>
+	<!-- ==== TRIP  DEBIT ================================================ -->
+	<xsd:element name="TripDebit" abstract="false" substitutionGroup="FareDebit_DummyType">
+		<xsd:annotation>
+			<xsd:documentation>A LOG ENTRY recording data about a debiting action for post-payment for a specific trip.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="TripDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="TripDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="TripDebitIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="TripDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for TRIP DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="FareDebit_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="TripDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="TripDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for TRIP DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence/>
```

Comment:
What is this?

Reply:
from TM: _A LOG ENTRY recording data about a debiting action for post-payment for a specific trip._
To be detailed in the doc